sdk/cosmosdb/cosmos/src/utils/batch.ts

```diff
@@ -210,6 +212,53 @@ export function decorateOperation(
   return operation as Operation;
 }
 
+/**
+ * Splits a batch into array of batches based on cumulative size of its operations by making sure
+ * cumulative size of an individual batch is not larger than {@link Constants.DefaultMaxBulkRequestBodySizeInBytes}.
+ * If a single operation itself is larger than {@link Constants.DefaultMaxBulkRequestBodySizeInBytes}, that
+ * operation would be moved into a batch containing only that operation.
+ * @param originalBatch - A batch of operations needed to be checked.
+ * @returns
+ * @hidden
+ */
+export function splitBatchBasedOnBodySize(originalBatch: Batch): Batch[] {
+  if (originalBatch?.operations === undefined && originalBatch.operations.length < 1) return [];
```

Comment:
Should this be an OR rather than AND? If no operations are defined OR the operations are of length zero? This condition seems like it will have the opposite of the intended effect because if `originalBatch?.operations` is undefined then the next condition must throw an error.